src/Microsoft.FeatureManagement/FeatureManagementOptions.cs

```diff
@@ -13,5 +13,11 @@ public class FeatureManagementOptions
         /// If missing feature filters are not ignored an exception will be thrown when attempting to evaluate a feature that depends on a missing feature filter.
         /// </summary>
         public bool IgnoreMissingFeatureFilters { get; set; }
+
+        /// <summary>
+        /// Controls the behavior of feature evaluation when the dependent feature declaration is missing.
```

Comment:
Controls the behavior of feature evaluation when the specified feature does not exist.
If missing features are not ignored, an exception will be thrown when attempting to evaluate them.